collection-scripts/gather_version

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# load shared functions and data when not sourced
+if [[ -z "$DIR_NAME" ]]; then
+    CALLED=1
+    DIR_NAME=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+    source "${DIR_NAME}/common.sh"
+fi
+
+function ocp_version() {
+    echo "openstack-k8s-operators/openstack-must-gather"> "${BASE_COLLECTION_PATH}/version"
+    ocp_version="0.0.0-unknown"
+    if [[ -n ${OS_GIT_VERSION} ]]; then
+        ocp_version="${OS_GIT_VERSION}"
+    else
```

Comment:
I think you can entirely remove the `else` section (L15-L20), and write to `/version` the resulting output (either `$OS_GIT_VERSION` or `0.0.0-unknown`).

Reply:
Done